src/Okta.Sdk.IntegrationTests/AuthorizationServersScenarios.cs

```diff
@@ -0,0 +1,603 @@
+                Description = "Test Authorization Server",
+                Audiences = new string[] { "api://default" },
+            };
+            var testAuthorizationServerPolicy = new OAuthAuthorizationPolicy
+            {
+                Name = $"{SdkPrefix}:{nameof(CreateAuthorizationServerPolicy)}:Test Authorization Policy",
+            };
+            var createdAuthorizationServer = await testClient.AuthorizationServers.CreateAuthorizationServerAsync(testAuthorizationServer);
+            var createdAuthorizationPolicy = await testClient.AuthorizationServers.CreateAuthorizationServerPolicyAsync(testAuthorizationServerPolicy, createdAuthorizationServer.Id);
+
+            try
+            {
+                createdAuthorizationPolicy.Should().NotBeNull();
+                createdAuthorizationPolicy.Name.Should().Be(testAuthorizationServerPolicy.Name);
+            }
+            finally
+            {
+                await testClient.AuthorizationServers.DeleteAuthorizationServerPolicyAsync(createdAuthorizationPolicy.Id, createdAuthorizationPolicy.Id);
+                await testClient.AuthorizationServers.DeactivateAuthorizationServerAsync(createdAuthorizationServer.Id);
+                await testClient.AuthorizationServers.DeleteAuthorizationServerAsync(createdAuthorizationPolicy.Id);
+            }
+        }
+
+        [Fact(Skip = "Api validation failed: com.saasure.framework.exception.KeyStoreLimitExceededException (400, E0000001): Unable to create new authorization server. You cannot create authorization servers because signing keys could not be generated. To fix this issue, contact support.")]
+        public async Task UpdateAuthorizationServerPolicy()
+        {
+            var testClient = TestClient.Create();
+            var testAuthorizationServerName = $"{SdkPrefix}:{nameof(UpdateAuthorizationServerPolicy)}:Test Authorization Server ({TestClient.RandomString(6)})";
+
+            var testAuthorizationServer = new AuthorizationServer
+            {
+                Name = testAuthorizationServerName,
+                Description = "Test Authorization Server",
+                Audiences = new string[] { "api://default" },
+            };
+            var testAuthorizationServerPolicy = new OAuthAuthorizationPolicy
+            {
+                Name = $"{SdkPrefix}:{nameof(UpdateAuthorizationServerPolicy)}:Test Authorization Policy",
+            };
+            var testUpdatedAuthorizationServerPolicy = new OAuthAuthorizationPolicy
+            {
+                Name = $"{SdkPrefix}:{nameof(UpdateAuthorizationServerPolicy)}:Test Authorization Policy Updated",
+            };
+            var createdAuthorizationServer = await testClient.AuthorizationServers.CreateAuthorizationServerAsync(testAuthorizationServer);
+            var createdAuthorizationPolicy = await testClient.AuthorizationServers.CreateAuthorizationServerPolicyAsync(testAuthorizationServerPolicy, createdAuthorizationServer.Id);
```

Comment:
Do we have an alias for this method? In other words, is it possible to do `createdAuthorizationServer.CreatePolicyAsync(testAuthorizationServerPolicy)` ?

Reply:
Wasn't certain if that was in scope for this set of changes.  Seems reasonable, I'll add it if it isn't there already.